timesketch/frontend-ng/src/components/Scenarios/ContextCard.vue

```diff
@@ -0,0 +1,47 @@
+<!--
+Copyright 2023 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-card outlined class="mt-3 mx-3" v-if="activeContext.question">
+    <v-toolbar dense flat>
+      <strong>{{ activeContext.question.display_name }}</strong>
+      <v-spacer></v-spacer>
+      <v-btn small icon @click="$store.dispatch('clearActiveContext')" class="mr-1">
+        <v-icon>mdi-close</v-icon>
+      </v-btn>
+    </v-toolbar>
+    <v-divider></v-divider>
+    <div class="pa-4 markdown-body" v-html="toHtml(activeContext.question.description)" style="font-size: 0.9em"></div>
```

Comment:
Would it make more sense to show the approach text here instead of repeating the question?

Reply:
This is the description of the question that is in use. The idea is to evolve this "DFIQ card" to also show approaches etc. But that will be done later.